cpp/src/traversal/sssp.cu

```diff
@@ -314,7 +316,7 @@ gdf_error gdf_sssp(gdf_graph* gdf_G,
                 GDF_INVALID_API_CALL);
   }
 
-  gdf_error err = gdf_add_adj_list(gdf_G);
+  /*gdf_error err = gdf_add_adj_list(gdf_G);
```

Comment:
-> Remove adding of adjacency list (added in sssp wrapper) and int32 checks (ensured when adding graph data)

Reply:
it seems we should keep GDF_REQUIRE(gdf_G->adjList, GDF_INVALID_API_CALL);